lib/__tests__/factory-sample-app.test.ts

```diff
@@ -7,3 +7,21 @@ describe('Factory.build', () => {
     expect(user.name).toEqual('susan');
   });
 });
+
+describe('Factory.rewindSequence', () => {
```

Comment:
Do these tests add additional confidence? They feel duplicative of the tests in [`factory.rewindSequence`](lib/__tests__/factory.test.ts). At a minimum, I think we could consider reducing this to a single test.

Reply:
@emilford , I don't disagree. 
My rationale here was to mimic how it might be used in an application. Which was based on the name of the test file.

What do you think?

Reply:
I agree that we these tests feel a bit duplicative. The tests you wrote in `factory.test.ts` are great and give enough confidence for me.